### ISSUE TITLE
Failed to load the initial page

### ISSUE THREAD

**Issue by sunyuhang** [vyakymenko/angular-seed-express#44]
**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**

Get the latest from Repo. npm install and then npm start. It stops at loading. It failed to load rxjs/observable package. Notice, the code seems do not have the latest changes from its parent repo. In its parent repo, it adds an new task to temporarily fix an angular issue: https://github.com/mgechev/angular-seed/blob/master/tools/tasks/seed/build.bundle.rxjs.ts
 
**Expected behavior**

It should continue to display the starting page
**Minimal reproduction of the problem with instructions**

It should continue to display the starting page
**What is the motivation / use case for changing the behavior?**

It should continue to display the starting page
**Please tell us about your environment:**

I use MAC Book.
* **Angular Seed Version:** `aaaaf75`

f614eec7af8da35be83e5191b27fa5bf0138a50b
* **Node:** `node --version` = 6.3.0

**Comment by vyakymenko** [comment]
@sunyuhang, thanks for an issue, fixed.

**Comment by sunyuhang** [comment]
Thanks.
On server side code, I can not use the map operator io rxjs. It is important for my project to use rxjs in express server. Please advice how to do this

Thanks very much

Sent from my iPhone

On Jul 17, 2017, at 2:37 AM, Valentyn Yakymenko <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #44<https://github.com/vyakymenko/angular-seed-express/issues/44>.

**Comment by sunyuhang** [comment]
Never mind. It is working now.
Thanks

Sent from my iPhone

On Jul 17, 2017, at 2:37 AM, Valentyn Yakymenko <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #44<https://github.com/vyakymenko/angular-seed-express/issues/44>.

**Comment by vyakymenko** [comment]
Sounds good!

Thanks!
